_posts/2022-04-08-utah-sgid-statewide-roads-data-layer-updates-04082022.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Utah SGID Statewide Roads Data Layer Updates 04/08/2022
+author:
+  display_name: Greg Bunce
+  email: gbunce@utah.gov
+date: 2022-04-08 7:30:00
+categories:
+  - Featured
+tags:
+  - Roads
+---
+
+UGRC recently completed updates to the SGID Roads feature class. Please visit our [Roads and Highway System]({% link data/transportation/roads-system/index.html %}) data page where you will find information about the Roads data model, as well as direct download links to the SGID Roads data in shapefile and geodatabase format.
+
+These updates are also reflected in UGRC's [address locators]({% link data/address-geocoders-locators/index.html %}), Utah's [Next-Generation 911 GIS database]({% link data/911/index.html %}), and UGRC's [street network analysis datasets]({% link data/transportation/street-network-analysis/index.html %}).
+
+The following are highlights from this month's update.
+
+#### County Updates
+
+New roads were added and road names and address ranges were updated for the following counties:
+
+- **Box Elder County:** Obtained roads data on 03/15/2022. Previous update was on 04/13/2021.
+- **Carbon County:** Obtained roads data on 02/15/2022. Previous update was on 11/09/2021.
+- **Davis County:** Obtained roads data on 03/15/2022. Previous update was on 03/10/2022.
+- **Kane County:** Obtained roads data on 03/15/2022. Previous update was on 11/09/2021.
+- **Millard County:** Obtained roads data on 01/27/2022. Previous update was on 01/12/2022.
+- **Salt Lake County:** Obtained roads data on 03/15/2022. Previous update was on 03/10/2022.
+- **Utah County:** Obtained roads data on . Previous update was on 03/10/2022.
```

Comment:
was this  meant to be empty?

Reply:
nope, good catch fixed in [this pr](https://github.com/agrc/gis.utah.gov/pull/1999)